### PR TITLE
(DOCSP-32259)(DOCSP-32262)(DOCSP-32264)(DOCSP-32265)(DOCSP-32266)(DOCSP-32293)(DOCSP-32263) Makes tools pages Atlas-centric

### PR DESCRIPTION
**Pull request by sarahsimpers**
## DESCRIPTION
Adds an Atlas-centric CTA to the top 250 pages in the database-tools docs repo

## STAGING
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/mongoimport/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/mongoexport/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/mongorestore/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/installation/installation-linux/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/installation/installation/
https://preview-mongodbsarahsimpers.gatsbyjs.io/database-tools/DOCSP-32259/installation/installation-windows/

## JIRA
https://jira.mongodb.org/browse/DOCSP-32259
https://jira.mongodb.org/browse/DOCSP-32262
https://jira.mongodb.org/browse/DOCSP-32264
https://jira.mongodb.org/browse/DOCSP-32263
https://jira.mongodb.org/browse/DOCSP-32265
https://jira.mongodb.org/browse/DOCSP-32266
https://jira.mongodb.org/browse/DOCSP-32293

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6500768324fcc731b469735b

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ianf-mongodb — COMMENTED** [review]
Hi @sarahsimpers, thanks for picking this up. I left a comment regarding this edit on the installation pages. Please let me know what you think!

File: source/installation/installation-linux.txt [hunk 1]
```diff
@@ -17,6 +17,10 @@ The {+dbtools+} are a suite of command-line utilities for working with
 MongoDB. Use this guide to install the {+dbtools-short+} on the Linux
 platform.
 
+.. |tool-command| replace:: the {+dbtools+}
```
[the remark this answers is not in the document]
Comment (sarahsimpers) [reply source/installation/installation-linux.txt:20]: Hi @ianf-mongodb, the intention for this CTA is to point out that database tools work with Atlas, but I will leave this open to the PM that will tech review whether we want to recommend something else over database tools.

**Review by ianf-mongodb — APPROVED** [review]
LGTM!

File: source/includes/fact-migrate-atlas.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+You can use |tool-command| to migrate from a self-hosted deployment
+to `MongoDB Atlas <https://www.mongodb.com/docs/atlas>`__. MongoDB
+Atlas is the fully managed service for MongoDB deployments in the
+cloud. To learn more, see `Seed with mongorestore 
+<https://www.mongodb.com/docs/atlas/import/mongorestore/>`__.
```
Comment (piakochar) [inline source/includes/fact-migrate-atlas.rst:5]: Should we also add a link to the documentation for Atlas-managed Migration services? https://www.mongodb.com/cloud/atlas/migrate or mongodb.com/docs/atlas/import/

It looks like the link here specifically directs users to using mongorestore to seed their db, but doesn't refer to C2C or managed Atlas migration at all

**Review by piakochar — APPROVED** [review]
LGTM, thank you!

**Pull request merged**